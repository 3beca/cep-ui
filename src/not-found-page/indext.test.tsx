import * as React from 'react';
import {render} from '@testing-library/react';
import {NotFoundPage} from './index';

test('NotFoundPage snapshot', () => {
    const {container, getByLabelText} = render(<NotFoundPage/>);
    getByLabelText(/page not found/i);
    expect(container).toMatchSnapshot();
});