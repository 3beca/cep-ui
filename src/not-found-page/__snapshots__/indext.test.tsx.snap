// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage snapshot 1`] = `
<div>
  <h1
    aria-label="page not found"
  >
    404 Page not found!
  </h1>
</div>
`;
